cpp/src/decisiontree/batched-levelalgo/builder_base.cuh

```diff
@@ -101,7 +101,7 @@ struct Builder {
   /** number of blocks used to parallelize column-wise computations. */
   int n_blks_for_cols = 10;
   /** Number of blocks used to parallelize row-wise computations. */
-  int n_blks_for_rows = 4;
+  int n_blks_for_rows = 40;
```

Comment:
even though this is (and was) hardcoded, in the ctor, can you compute this value based on the multi-processor count and kernel occupancy?

Reply:
sure thejaswi, completely forgot!